src/emcee/moves/red_blue.py

```diff
@@ -93,11 +93,15 @@ def propose(self, model, state):
             new_log_probs, new_blobs = model.compute_log_prob_fn(q)
 
             # Loop over the walkers and update them accordingly.
+            try:
```

Comment:
I would get rid of the try/except block (see below)

Reply:
you can use `model.random.uniform`, it works with `RandomState` and `Generator`.